src/css/support.js

```diff
@@ -29,6 +29,11 @@ define( [
 		var divStyle = window.getComputedStyle( div );
 		pixelPositionVal = divStyle.top !== "1%";
 
+		// Don't run until window is visible (https://github.com/jquery/jquery-ui/issues/2176)
+		if ( div.offsetWidth === 0 ) {
+			return;
```

Comment:
Please also run `documentElement.removeChild( container );` here to avoid leaving the container in the DOM.